### PR TITLE
samples(spanner): PITR samples backup fix

### PR DESCRIPTION
**Pull request by thiagotnunes**
Instead of using the earliest version time of the database, uses the current time (from spanner). If we used the earliest version time of the database instead we would be creating an empty backup, since the database we are backing up is not 1 hour old (default version retention period).

### CODE REVIEW

**Comment by thiagotnunes** [comment]
Test failures seem unrelated to the PR, they are failures for the `Storage.Samples.Tests` (not modified here).

File: spanner/api/Spanner.Samples.Tests/CreateBackupTest.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Google Inc.
+// Copyright 2021 Google Inc.
```
Comment (amanda-tarafa) [inline spanner/api/Spanner.Samples.Tests/CreateBackupTest.cs:1]: nit: Please revert this change, we don't update the copyright year when we make modifications, the code is copyrighted since it was first published.

**Review by amanda-tarafa — APPROVED** [review]
LGTM
I'm fine with this change, but I wonder if it's really necessary. We are not showing users where we are getting the version date from so for them is irrelevant. And for testing the sample itself , it's also irrelevant if we created the backup with either date.
If you are looking at consistency across languages then that's understandable.

(We have some flakiness in this repo that we are slowly fixing, don't worry about unrelated failures.)

**Comment by amanda-tarafa** [comment]
#1272 and #1274 for tracking flakiness seen on presubmits in this PR.

File: spanner/api/Spanner.Samples.Tests/CreateBackupTest.cs (hunk 1 above)
Comment (thiagotnunes) [reply spanner/api/Spanner.Samples.Tests/CreateBackupTest.cs]: Fixed.

**Comment by thiagotnunes** [comment]
@amanda-tarafa yes, you are correct, we are looking to have all the languages consistently use the `current_timestamp()` function.

**Review by skuruppu — APPROVED** [review]

**Comment by amanda-tarafa** [comment]
More flakiness tracked by #1277 and #1278. All failures are unrelated to this PR. Merging...
